.github/workflows/publish-solidity-sdk.yml

```diff
@@ -17,25 +17,31 @@ jobs:
           node-version-file: "package.json"
           registry-url: "https://registry.npmjs.org"
 
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          run_install: true
+
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1.1.1
         with:
           version: nightly
 
       - name: Install dependencies
-        run: npm install
-        working-directory: sdk/solidity/
+        run: pnpm --filter @pythnetwork/express-relay-sdk-solidity install
 
       - name: Generate ABIs
-        run: npm run generate-abi
-        working-directory: sdk/solidity/
+        run: pnpm generate-abi
```

Comment:
For this kind of thing I generally make the package that requires it have scripts that look like this:

```
"scripts": {
  "build": "pnpm build:abi && pnpm build:tsc",
  "build:abi": "...",
  "build:tsc": "tsc"
}
```

Of course in the crosschain repo turborepo takes care of the `build` script.

This way you don't have to configure CI workflows to be aware of the steps required to build individual packages.

Reply:
For now I just built all packages using `pnpm -r build` :D